db/migrate/20160324174132_create_deployments.rb

```diff
@@ -0,0 +1,23 @@
+class CreateDeployments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployments do |t|
+      t.string   :deployment_type
+      t.string   :deployment_version
+      t.boolean  :containerized, :default => true
+      t.string   :deployment_method
+      t.string   :deployed_provider_type
+      t.string   :deployed_provider_name
+      t.string   :ssh_private
+      t.string   :ssh_public
+      t.string   :ssh_user
+      t.string   :rhn_user
+      t.string   :rhn_pass
+      t.string   :rhn_sku
```

Comment:
@detiber @simon3z 
* I'll rename those variables.
* I'll leave the SKU in for now in the db, because that's the only way it works now, ill also add pool-id.